symphony/content/content.blueprintspages.php

```diff
@@ -649,7 +648,6 @@ public function __actionEdit()
 
             if (trim($fields['handle']) == '') {
                 $fields['handle'] = $fields['title'];
-                $autogenerated_handle = true;
```

Comment:
Look at the complete code path. Even if true, it will revert to false on line 680. 2 pages with the same name should not cause any problems, as long as their respective handles are different. I had to do it once and had to hack the DB.